### PR TITLE
Refactoring admin-page.js 

### PR DESCRIPTION
**Pull request by fructusmortus**
to meet it with redesigned admin-table in contacts-admin.ejs also fixed constants import

### CODE REVIEW

File: module-1/dist/assets/js/admin-page.js [hunk 1]
```diff
@@ -53,13 +53,21 @@ replyButtons.forEach(elem => {
...
                 // TODO process data
             })
             .catch((error) => {
                 throw new Error(error)
             })
+            form.reset()
+            form[0].value=""
+            form[0].placeholder = "Your mail has been sent"
+            form[0].classList.add('color__placeholder')
+            function formClose() {
```
Comment (shelby-carter) [inline module-1/dist/assets/js/admin-page.js:67]: Don't change styles in JS, instead add or remove certain css class and have an appropriate rules in css

File: module-1/dist/assets/js/admin-page.js [hunk 2]
```diff
@@ -53,13 +53,21 @@ replyButtons.forEach(elem => {
...
+                form.style.display = "none"
+            }
+            setTimeout(formClose, 3000)
```
Comment (shelby-carter) [inline module-1/dist/assets/js/admin-page.js:70]: Why do we need a timeout here? Timeouts should be avoided at any costs except very rare cases when they're absolutely necessary

File: module-1/dist/assets/js/admin-page.js [hunk 3]
```diff
@@ -53,13 +53,21 @@ replyButtons.forEach(elem => {
...
         })
     })
 })
```
Comment (shelby-carter) [inline module-1/dist/assets/js/admin-page.js:73]: Newline missing

File: module-1/index.js [hunk 4]
```diff
@@ -291,7 +291,7 @@ app.delete('/deleteContactFormData/:id', async (req, res) =>{
 
 app.post('/sendEmail', async (req, res) =>{
 
-  const replyEmail = 'serega.umerenkov@gmail.com'
+  const replyEmail = 'fructusmortus@gmail.com'
```
Comment (shelby-carter) [inline module-1/index.js:294]: Don't hardcode emails in the code, move to constants, name it `CONTACT_FORM_REPLY_EMAIL` and import in this file

File: module-1/views/contacts-admin.ejs [hunk 5]
```diff
@@ -36,7 +34,7 @@
 		<%})%>
 	</div> -->
 	
-	<div class="table">
+	<div id="aaa" class="table">
```
Comment (shelby-carter) [inline module-1/views/contacts-admin.ejs:37]: `aaa` is not an appropriate class name, use something meaningful

File: module-1/views/contacts-admin.ejs [hunk 6]
```diff
@@ -46,18 +44,19 @@
...
-					<textarea class="table__textarea" placeholder="Whrite your message"></textarea> 
+					<textarea class="table__textarea" name="message" placeholder="Whrite your message"></textarea> 
 					<button class="textarea__btn" type="submit">Send</button>
 				</form>
 			</div>
 		<%})%>
 	</div>	
 </body>
+<script type="module" src="/assets/js/admin-page.js"></script>
 </html>
```
Comment (shelby-carter) [inline module-1/views/contacts-admin.ejs:62]: Newline

File: module-1/dist/assets/css/style-contacts.css [hunk 7]
```diff
@@ -433,6 +80,12 @@ dropdown.closed {
   resize: none;
 }
 
+.color__placeholder::placeholder {
+  font-size: 19px;
+  color: #f78707;
```
Comment (shelby-carter) [inline module-1/dist/assets/css/style-contacts.css:85]: Don't use colours directly, move all colours to a variables in a separate file, call it `_colors.scss` and use it this way. Also why we using plain css here? It must be `SASS`, nobody uses vanilla css anymore. Please discuss with @USergei how to fix build scripts that were originally using SASS

File: module-1/dist/assets/css/style-contacts.css [hunk 8]
```diff
@@ -433,6 +80,12 @@ dropdown.closed {
...
+  text-align: center; 
+}
+
 .table__textarea:focus {
   border: 0.5px solid #353333;
```
Comment (shelby-carter) [inline module-1/dist/assets/css/style-contacts.css:90]: Same for all css colours as per comment above

File: module-1/dist/assets/js/admin-page.js [hunk 9]
```diff
@@ -40,10 +38,12 @@ async function postData(url = '', data = {}) {
 
 replyButtons.forEach(elem => {
     elem.addEventListener("click", event => {
-        let text = event.target.closest("div.td").querySelector('textarea')
-        let form = event.target.closest("div.td").querySelector('form')
-        let email = event.target.parentNode.parentNode.querySelector('.email').innerText
-        let message = event.target.parentNode.parentNode.querySelector('.message').innerText
+        let parent = event.target.closest(".table__row")
```
[the remark this answers is not in the document]
Comment (fructusmortus) [reply module-1/dist/assets/js/admin-page.js:41]: Done

File: module-1/dist/assets/js/admin-page.js (hunk 1 above)
Comment (fructusmortus) [reply module-1/dist/assets/js/admin-page.js:67]: Maybe, I don't understand correctly but we did as you describe. Adding a style at the string 66, 'color__placeholder' is a certain class in the style-contacts.css

File: module-1/dist/assets/js/admin-page.js (hunk 2 above)
Comment (fructusmortus) [reply module-1/dist/assets/js/admin-page.js:70]: I'll take a note. Hovever, to solve your task about demonstration of an approval after a mail exchange, we have come up only with that solution. Could you demonstrate some alternatives to us during today's lesson?

File: module-1/dist/assets/js/admin-page.js (hunk 3 above)
Comment (fructusmortus) [reply module-1/dist/assets/js/admin-page.js:73]: Fixed

File: module-1/views/contacts-admin.ejs (hunk 6 above)
Comment (fructusmortus) [reply module-1/views/contacts-admin.ejs:62]: Fixed

File: module-1/views/contacts-admin.ejs (hunk 5 above)
Comment (fructusmortus) [reply module-1/views/contacts-admin.ejs:37]: Deleted

File: module-1/index.js (hunk 4 above)
Comment (fructusmortus) [reply module-1/index.js:294]: Done

**Pull request merged**